### PR TITLE
Issue 2108: The test that checks packages stored in Gremlin should have better logic for the latest_version property

### PR DESCRIPTION
**Pull request by tisnik**
Original issue:
https://github.com/openshiftio/openshift.io/issues/2108

### CODE REVIEW

File: integration-tests/features/src/attribute_checks.py [hunk 1]
```diff
@@ -185,3 +185,9 @@ def check_cve_value(cve, with_score=False):
     if with_score:
         score = float(match.group(2))
         assert score >= 0.0 and score <= 10.0
+
+
+def parse_version(value):
+    """Parse the version string and split it into major, minor, patch numbers."""
+    match = re.match("(\d)\.(\d)\.(\d).*", value)
+    return int(match.group(1)), int(match.group(2)), int(match.group(3))
```
Comment (jpopelka) [inline integration-tests/features/src/attribute_checks.py]: or by using [semantic_version](https://pypi.python.org/pypi/semantic_version/)
```python
from semantic_version import Version
v = Version(value)
return v.major, v.minor, v.patch
```

File: integration-tests/features/steps/gremlin.py [hunk 2]
```diff
@@ -329,6 +329,34 @@ def check_property_value(context, property_name, expected_value):
...
+
+    if major == expected_major:
+        # major versions are the same -> 2nd step - compare minor versions
+        assert minor >= expected_minor, "Minor version {v} is less then {e}".format(
+            v=minor, e=expected_minor)
+
+        if minor == expected_minor:
+            # minor versions are the same -> 3nd step - compare patch versions
+            assert patch >= expected_patch, "Patch version {v} is less then {e}".format(
+                v=patch, e=expected_patch)
```
Comment (jpopelka) [inline integration-tests/features/steps/gremlin.py]: or
```python
from semantic_version import Version
assert Version(value) > Version(expected_value)
```

File: integration-tests/features/src/attribute_checks.py (hunk 1 above)
Comment (tisnik) [reply integration-tests/features/src/attribute_checks.py]: good. sure, let's try this

Comment (jpopelka) [reply integration-tests/features/src/attribute_checks.py]: Well, I think you actually don't need it at all, see my other comment :-)

**Pull request merged**